components/authz-service/storage/postgres/migration/sql/44_policy_rules.up.sql

```diff
@@ -0,0 +1,71 @@
+BEGIN;
+
+CREATE TYPE iam_rule_type AS ENUM ('node', 'event');
+CREATE TYPE iam_attributes AS ENUM ('organization', 'chef-server',
+  'environment', 'role', 'chef-tag', 'policy-name', 'policy-group');
+CREATE TYPE iam_condition_operator AS ENUM ('equals', 'member-of');
+
+CREATE TABLE iam_project_rules (
+  db_id SERIAL PRIMARY KEY,
+  id TEXT NOT NULL UNIQUE,
+  project_id TEXT REFERENCES iam_projects ON DELETE CASCADE,
+  name TEXT NOT NULL,
+  type iam_rule_type NOT NULL
+);
+
+CREATE TABLE iam_rule_conditions (
+  db_id SERIAL PRIMARY KEY,
+  rule_id INTEGER REFERENCES iam_project_rules ON DELETE CASCADE,
+  value TEXT NOT NULL,
```

Comment:
I thought value could sometimes be an array, depending on the operator. We should confirm.

Reply:
I see later that we're using a string-- I'm curious about the choice if we have time to chat about that!

Reply:
Not clear what the proper value is yet, but if we did wanna go the array route for `member-of` type conditions, we'll have to rethink the schema a bit, since that'd be different than what we wanted for `equals.